udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -20,13 +20,55 @@ async function deleteCommentData() {
   await getCommentData;
   window.location.reload();
 }
+
 /** Creates an <li> element containing text. */
 function createCommentListItem(inputText) {
   const listElement = document.createElement('li');
   listElement.innerText = inputText;
   return listElement;
 }
 
+/** Creates a map to show University. */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 32.880406, lng: -117.242677}, zoom: 16});
+
+  const trexMarker = new google.maps.Marker({
+    position: {lat: 32.880406, lng: -117.242677},
+    map: map,
+    title: 'UC San Diego'
+  });
+}
+
+google.charts.load('current', {'packages': ['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Fetches skill data and uses it to create a chart. */
+function drawChart() {
+  fetch('/endorse-data')
+      .then(response => response.json())
+      .then((skillVotes) => {
```

Comment:
I'm having a hard time figuring out what kind of values `skillVotes` contains.

It'd be helpful for me if there were comments about what it contains.

My guess is that this is an array of objects having a key == skill and object == number of votes, but I'm not sure.

Reply:
Done

Reply:
Maybe add here:

```
The skillVotes variable is an array of objects that each have a single key-value pair where the key is a string representing a skill and the value is a number representing the number of endorsements received for that skill.
```